test/event_handler_test.py

```diff
@@ -3,24 +3,57 @@
 import io
 import json
 import uuid
+import psycopg2
 from moto import mock_sqs
 from unittest import TestCase
 from unittest.mock import patch
+from datetime import datetime
 
 from src import event_handler
+from src.db_helper import RunInTransaction
 
+EVENT_TYPE = 'session_event'
+TIMESTAMP = '2018-02-10 12:07:32'
+SESSION_EVENT_TYPE = 'success'
 
 @mock_sqs
 class EventHandlerTest(TestCase):
```

Comment:
Do we need to execute these test cases locally (without Docker)?

Reply:
You'd need to spin up your own postgres DB.
The easiest way to do this is using Docker, in which case you are back to square one.

It would be quite nice to run postgres in docker, and then run tests locally (read in your IDE) and connect to postgres in Docker.
This is definitely on my list of 'nice to haves' that I'll be working on in a bit. (we should be able to tweak the Docker Image to allow this)

Reply:
Note we got mongodb-in-docker to work for hub-support tests, even when running tests outside docker - I'd hope we could do this for postgres as well.

Reply:
yep - see https://github.com/alphagov/verify-event-recorder-service/pull/10